cogeo_mosaic/backends/base.py

```diff
@@ -24,35 +25,66 @@
 from cogeo_mosaic.utils import bbox_union
 
 
+def update_zoom_and_bounds(instance, attribute, value):
```

Comment:
magic method that get's called every time `mosaic_def` is set (updating zooms and bounds)